pokedex-with-bit/pods/doc.yml

```diff
@@ -1,6 +1,17 @@
-!DataURIPbIndexer
-with:
-  index_filename: doc.gzip
+!CompoundIndexer
+components:
+  - !NumpyIndexer
+    with:
+      index_filename: vec.gz
+    metas:
+      name: vecidx  # a customized name
+      workspace: $WORKDIR
+  - !DataURIPbIndexer
```

Comment:
I think the problem comes from this, if it works alone it would have the proper Drivers attached, but working in a CompoundIndexer it may not, you can take inspiration on CrossModal Search, in the ImageIndexer there is the usage of `BinaryPbIndexer` that uses as one of its Drivers a Buffer2URI {}